packages/foundry-sdk-generator/src/generate/commandUtils.ts

```diff
@@ -22,3 +22,371 @@ export function transformArrayArg(arg?: string[]) {
   }
   return arg;
 }
+
+export const osdkTypes = new Set([
```

Comment:
Literally just copied and pasted what we export from the root of legacy client, since we will start exporting all the 2.0 types from root in 1.3 version of OSDK

Reply:
nit: select lines, cmd+p, "sort"

Reply:
Done